### ISSUE TITLE
No tray icon in version 2.0.10. Continues to run in background after quitting. Linux.

### ISSUE THREAD

**Issue by RasheedAZ** [FilenCloudDienste/filen-desktop#123]
On version 2.0.10 I no longer have the tray icon. Also, after choosing quit it continues to run in the background.

_Originally posted by @joshuarcase in https://github.com/FilenCloudDienste/filen-desktop/issues/56#issuecomment-1340324207_
      
I have the same issue.

Pop!_OS 22.04 with Gnome 42.3.1

**Comment by joshuarcase** [comment]
I meant it continues to run in the background without a tray icon even after choosing quit from the in-app menu. I have to use pkill to stop it.

**Comment by DanielProg39** [comment]
"No tray icon issue" - it seems to be not a bug, but "feature", because it was removed by developers due to problems some people experienced with it (see #56 for more). I did't have any of them, it was working perfectly for me, so I see two workarounds: 1. As Jan previously proposed, to add a setting to enable tray icon (with warning that it might be not stable); 2. The best way, as I think, is to port an application to Flatpak.
As seen in #9 there has been some work on porting the application to Flatpak, but it was half a year ago. I don't know how things are going now.
